src/index.js

```diff
@@ -0,0 +1,237 @@
+  }
+
+  todos.forEach((todo) => {
+    const card = document.createElement('li');
+    const cbContainer = document.createElement('div');
+    const cbInput = document.createElement('input');
+    const check = document.createElement('span');
+    const item = document.createElement('p');
+    const editInput = document.createElement('input');
+    const button = document.createElement('span');
+    const edit = document.createElement('span');
+    const icon = document.createElement('i');
+    const iconEdit = document.createElement('i');
+    card.classList.add('card');
+    button.classList.add('clear');
+    edit.classList.add('editInputShow');
+    edit.classList.add('edit-btn');
+    cbContainer.classList.add('cb-container');
+    cbInput.classList.add('cb-input');
+    item.classList.add('item');
+    editInput.classList.add('editInput');
+    check.classList.add('check');
+    button.classList.add('clear');
+    icon.classList.add('fa', 'fa-times');
+    iconEdit.classList.add('fas', 'fa-ellipsis-v');
+
+    card.setAttribute('draggable', true);
+    card.setAttribute('data-id', todo.id);
+    cbInput.setAttribute('type', 'checkbox');
+    item.textContent = todo.item;
+
+    editInput.setAttribute('type', 'text');
+    editInput.setAttribute('data-id', todo.id);
+    editInput.setAttribute('value', todo.item);
+    editInput.setAttribute('class', 'editInput');
+    if (todo.completed) {
+      card.classList.add('checked');
+      cbInput.setAttribute('checked', 'checked');
+    }
+
+    card.addEventListener('dragstart', function () {
+      this.classList.add('dragging');
+    });
+    card.addEventListener('dragend', function () {
+      this.classList.remove('dragging');
+    });
+
+    cbInput.addEventListener('click', function () {
+      const correspondingCard = this.parentElement.parentElement;
+      const { checked } = this;
+      stateTodo(
+        [...document.querySelectorAll('.todos .card')].indexOf(
+          correspondingCard,
+        ),
+        checked,
+      );
+      checked
+        ? correspondingCard.classList.add('checked')
+        : correspondingCard.classList.remove('checked');
+    });
+
+    button.addEventListener('click', function () {
+      const correspondingCard = this.parentElement;
+      correspondingCard.classList.add('fall');
+      removeTodo(
+        [...document.querySelectorAll('.todos .card')].indexOf(
+          correspondingCard,
+        ),
+      );
+      correspondingCard.addEventListener('animationend', () => {
+        setTimeout(() => {
+          correspondingCard.remove();
+        }, 100);
+      });
+    });
+
+    button.appendChild(icon);
+    edit.appendChild(iconEdit);
+    cbContainer.appendChild(cbInput);
+    cbContainer.appendChild(check);
+    card.appendChild(cbContainer);
+    card.appendChild(item);
+    card.appendChild(editInput);
+    card.appendChild(button);
+    card.appendChild(edit);
+    document.querySelector('.todos').appendChild(card);
+  });
```

Comment:
Example for Modularity:

The Function passed to this event Listener can be in another File and you can import it.

I am not saying do this for everything but the functions that are too long can be extracted to a separate file or small chunks of code inside this function can be extracted to a separate Function to make this Function shorter and more readable.

Reply:
Thank you so much @awais-amjed  I take note and I will work on this.